.github/workflows/release.yml

```diff
@@ -220,3 +220,9 @@ jobs:
         run: pnpm --recursive publish --tag ${{ inputs.release_channel }} --no-git-checks
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Upload Standalone Artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: tailwindcss-standalone
+          path: packages/@tailwindcss-standalone/dist/
```

Comment:
This will only be saved to the workflow run for now but that's probably fine and should be easy to change to be added to the GitHub release once we have it.